meta-refkit/conf/distro/refkit.conf

```diff
@@ -307,3 +307,7 @@ INHERIT += "uninative"
 # stage due to lack of RTC backup battery. This causes unnecessary fixes being
 # made due to filesystem metadata time stamps being in future.
 APPEND_append = " fsck.mode=skip"
+
+KERNEL_FEATURES_append = "${@ bb.utils.contains('DISTRO_FEATURES', 'dm-verity', ' dm-verity.scc', '', d) }"
+SRC_URI_append_pn-linux-yocto = " file://dm-verity.scc"
+FILESEXTRAPATHS_prepend_pn-linux-yocto = "${META_REFKIT_BASE}/files/linux-yocto:"
```

Comment:
@pohly linux-intel is what we get from meta-intel nowadays. Have you submitted these in yocto-kernel-cache already? We've targeted to not have BSP bbappends here but all comes from meta-intel.